Dizzy/Screens/Login/LoginVM.swift

```diff
@@ -30,7 +30,7 @@ protocol LoginVMNavigationDelegate: class {
     func userLoggedIn(user: DizzyUser)
     func navigateToSignInScreen()
     func navigateToAppInfoScreen(type: AppInfoType)
-    func navigateToAdminScreen()
+    func navigateToAdminScreen(with user: DizzyUser)
```

Comment:
we are registrating  the logged in user and every entity can ask for it as a dependency in it's init
so you dont . need to transfer it